### DISCUSSION TITLE
Strange delay before playing a sound after another sound has finished playing

### DISCUSSION THREAD

**Discussion by Lebostein** [Makuna/DFMiniMp3#129] [Q&A]
I have s simple circuit with an Arduino Nano, a DFPlayer and 3 buttons. Button 1 calls **playMp3FolderTrack(1)**, Button 2 calls **playMp3FolderTrack(2)** and button 3 calls **stop()**. Sounds 1 and 2 are very short (3 seconds each).

If I press button 1, the sound 1 is played immediately without delay. If the sound has finished and I press button 1 again or button 2 then the next sound is played with a delay of 0,5 - 1 seconds. Very anoying.

But: If I press button 3 (=stop) after a sound has finished playing, then the next sound is played immediately without delay, like the first after starting the program.

What is going on?

**Comment by Makuna** [comment]
Reminder, issues area is for tracking bugs, [this part of the wiki](https://github.com/Makuna/DFMiniMp3/wiki#seeking-help) directs you to discussion area where this now was moved.

Sample Sketch that demonstrates what you are seeing?

Turn on [Debug Feature](https://github.com/Makuna/DFMiniMp3/wiki/Debug-Features) and include the output.
